Update README and Contributing guide (#937)
rudderlabs/rudderstack-docs: CONTRIBUTING.mdx

We prefer squash or rebase commits so that all changes from a branch are committed to master as a single commit. All pull requests are squashed when merged, but rebasing prior to merge gives you better control over the commit message.

> ## Build the docs

Reviewer: ```suggestion
## Building the docs
```

> To build docs locally, you should have a minimum [Gatsby](https://www.gatsbyjs.com/) version of 4, and a fairly recent version of Node installed.

Reviewer: ```suggestion
We highly recommend building the docs locally to preview and test your changes before pushing them to your branch. To do so, you should have [Gatsby](https://www.gatsbyjs.com/) version 4 or above and a fairly recent version of Node.js installed on your system.
```

> 1. Clone this repo.

Reviewer: ```suggestion
Follow these steps to locally build the RudderStack docs site:

1. Clone this repository.
```

> 1. Install dependencies with `yarn install`.

Reviewer: ```suggestion
2. Install the necessary dependencies with `yarn install`.
```

> 1. Build a local version with `yarn start`.

Reviewer: ```suggestion
3. Build a local version with `yarn start`.
```

> 1. View your RudderStack Docs build at `http://localhost:8001/`.

Reviewer: ```suggestion
4. View your RudderStack docs build at `http://localhost:8001/`.
```

## Getting help

> If you have any questions, concerns, or queries, you can start by asking a question in our [**Slack**](https://rudderstack.com/join-rudderstack-slack-community/) community.

Reviewer: ```suggestion
If you have any questions or concerns, you can start a conversation in our [**Slack**](https://rudderstack.com/join-rudderstack-slack-community/) community.
```
